src/versionbits.cpp

```diff
@@ -112,8 +112,12 @@ BIP9Stats AbstractThresholdConditionChecker::GetStateStatisticsFor(const CBlockI
     stats.period = Period(params);
     stats.threshold = Threshold(params);
 
-    if (pindex == NULL)
+    if (pindex == NULL) {
+        stats.elapsed = 0;
```

Comment:
I see no problem in:
 - moving all default values to `BIP9Stats` constructor, or
 - moving these up even if they are overwritten below.